dist/yaml/ovnkube-master.yaml

```diff
@@ -47,6 +47,8 @@ spec:
         # This is the community build
         image: "docker.io/ovnkube/ovn-daemonset:latest"
         # This is an official redhat build
```

Comment:
We should remove the official red hat build stuff.

Reply:
@rajatchopra would it be possible to leave the RH in? This is very convenient for me when testing on my cluster, I just adjust the commented lines.

Reply:
No problem with the meta data in comments I think.